src/js/view/controls/components/timeslider.js

```diff
@@ -196,7 +196,7 @@ class TimeSlider extends Slider {
         if (duration === 0) {
             this._api.play(reasonInteraction());
         } else if (this.streamType === 'DVR') {
-            const seekRange = this._model.get('seekRange');
+            const seekRange = this._model.get('seekRange') || { start: 0 };
```

Comment:
Do you think it would be better to set seekRange to this default in `program-listeners` instead?

Reply:
In that case I would expect `start` and `end` to be defined, and that would require a lot more work. I went this route because it's the simplest change and `seekRange` reflects the output of provider "time" and "meta" events.